config/routes.rb

```diff
@@ -1,3 +1,14 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
```

Comment:
Favor the use of routes for used actions, e.g.: `resources :sessions, only: [:new, :create, :destroy]`. [OPTIONAL]